WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV0.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+class ConversationsAPIV0: ConversationsAPI, VersionedAPI {
+
+    // MARK: - Constants
+
+    enum Constant {
```

Comment:
```suggestion
    private enum Constants {
```
nitpic: but what we stick with plural or singular on ConnectionsAPI it's plural and private

Reply:
Me personally prefer singular when you use it it's `Constant.batchSize`. But I can change it to plural again. It can't be private, because `ConversationsAPIV1` uses the constants too.